orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/DetermineRollbackCandidatesTask.java

```diff
@@ -163,28 +163,28 @@ public TaskResult execute(@Nonnull Stage stage) {
             cluster.get("serverGroups"), new TypeReference<List<ServerGroup>>() {});
     serverGroups.sort(Comparator.comparing((ServerGroup o) -> o.createdTime).reversed());
 
+    List<ServerGroup> enabledServerGroups =
```

Comment:
LGTM - so by filtering disabled server groups in `serverGroupsInRegion` , we are making sure to rollback only to an enable server group?

Reply:
Yep, that's it!